plugins/dependencies/src/main/resources/refreshVersions-rules/androidx-version-alias-rules.txt

```diff
@@ -82,9 +82,6 @@ androidx.lifecycle:lifecycle-viewmodel-compose
 androidx.media:media-widget(-*)
 ^^^^^^^^.      ^^^^^^^^^^^^
 
-androidx.navigation:navigation-compose
```

Comment:
Should this have been kept?

Reply:
My mistake. Just saw https://github.com/jmfayard/refreshVersions/pull/550#event-6922140048